src/components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <input type="text" className="input" readOnly value={0} />
+
+    <div className="buttons">
+      <div className="row-one">
+        <button className="button" type="button">AC</button>
+        <button className="button" type="button">+/-</button>
+        <button className="button" type="button">%</button>
+        <button className="button operator" type="button">/</button>
```

Comment:
- [ ] The division sign on the design is different from the one you have. Kindly use the correct sign: "&div;". You can use this: `&div;` if you do not have it. This is required for future projects.